pages/client-libraries/nodejs.mdx

```diff
@@ -550,6 +550,62 @@ a transaction to encapsulate the Cypher queries you're executing. This automatic
 transaction management simplifies the process of working with transactions. 
 That method should be used when transaction control is unnecessary. 
 
+##### Process partial results
+
+When dealing with large queries or queries that may take considerable time to
+execute, it's possible to handle partial results before the query reaches the
+timeout. This is useful in scenarios where timeouts or other issues might
+interrupt the execution.
+
+In the example below, due to the short timeout set on the query, it's likely
+(and expected) that the query will be interrupted before completion. Therefore,
+it's useful to handle the partial results that may be returned before the
+timeout occurs.
```

Comment:
This is good, but is it okay to say that this is also great in cases when you have too many records being returned and you want to get partial results to forward on your frontend or similar? You can do other stuff in the app while waiting for the full result back. 

> "The advantage of the use of the reactive drivers is that gives the flexibility to the client to request data with specified size until they are totally consumed or terminate the transaction when needed."

Why use reactive API?
Usually,  if using `await` with a query, the application waits for the server to retrieve all query results and transmit them to the driver. This is not a problem for most use cases, but for queries that have a long processing time or a large result set, asynchronous handling may speed up the application. -> https://neo4j.com/docs/javascript-manual/current/query-async/#_asynchronous_iteration